app/scrapers/parsijoo.py

```diff
@@ -29,3 +31,42 @@ def parse_response(soup):
         print('Parsijoo parsed: ' + str(urls))
 
         return urls
+
+    @staticmethod
+    def parse_video_response(soup):
+        """ Parse response and returns the urls
+
+            Returns: urls (list)
+                    [[Tile1, url1], [Title2, url2], ...]
+        """
+        urls = []
+        for a in soup.findAll('a', attrs={'class': 'over-page'}):
+            title = a.get('title')
+            url = 'https://video.parsijoo.ir' + a.get('href')
+            urls.append({
+                'title': title,
+                'link': url
+            })
+
+        print('Parsijoo parsed: ' + str(urls))
+
+        return urls
+
+    @staticmethod
+    def parse_image_response(soup):
+        """ Parse response and returns the urls
+
+            Returns: urls (list)
+                    [[url1], [url2], ...]
+        """
+        urls = []
+        for div in soup.findAll('div', attrs={'class': 'image-container overflow'}):
```

Comment:
for div in soup.find_all('div', class_='image-container overflow'):  # for PEP8 compatibility